posts/templates/post/post.html

```diff
@@ -9,17 +9,18 @@
     <link rel="stylesheet" href="{% static 'post/post.css' %}">
     <script src="{% static 'post/post.js' %}"></script>
 </head>
-<body>
+<body onload="generateAuthorLinks();">
 <div class="container-fluid">
+    {% load extra_filters %}
     <div id={{post.id}} class="post-container container-fluid disabled">
         <div class="row">
             <div class="post-author col-md-8">
-                <h5>{{ post.author.displayName }}</h5>
+                <h5 class="author-link" data-user="{{ post.author.id|parse_id }}">{{ post.author.displayName }}</h5>
```

Comment:
basically any tag that has:
- the author-link class
- data-user defined as a user id
will make that tag linked to the users feed.
Also it needs:
- generateAuthorLinks() to be called somewhere in order to convert those elements to links
- and the static/post/post.js to be imported in order to use generateAuthorLinks()